Standards/scs-0103-v1-standard-flavors.md

```diff
@@ -0,0 +1,145 @@
+---
+title: SCS Standard Flavors and Properties
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+## Motivation
+
+In OpenStack environments there is a need to define different flavors for instances.
+The flavors are pre-defined by the operator, the customer can not change these.
+OpenStack providers thus typically offer a large selection of flavors.
+
+While flavors can be discovered (`openstack flavor list`), it is helpful for users (DevOps teams),
+to have a guaranteed set of flavors available on all SCS clouds, so these need not be discovered.
+
+## Properties (extra specs)
+
+The following extra specs are recognized, together with the respective semantics:
+
+- `scs:name-vN=NAME` (where `N` is `1` or `2`, and `NAME` is some string) means that the
+  flavor is one of the
+  standard SCS flavors, and the requirements of Section "Standard SCS flavors" below apply.
+- `scs:cpu-type=shared-core` means that _at least 20% of a core in >99% of the time_,
+  measured over the course of one month (1% is 7,2 h/month).
+- `scs:diskN-type=ssd` (where `N` is a nonnegative integer, usually `0`) means that the
```

Comment:
Likewise, we should spell out the allowable `disk`N`-type` settings, no?
`network` (`n`), `hdd` (`h`), `ssd` (`s`), `nvme` (`p`) and also mention that is is allowable to advertize NVMes as SSDs (as understatement is allowable).

We could of course reference another standard here, if we prefer so.
(And as we have not yet a complete list of extra specs, maybe we do two sets of them to have the first one ready quickly?)

Reply:
See above. Yes, we can go ahead and standardize the extra specs fully, but I would rather do one step at a time. What we need is a proof of concept that discoverability in this way does indeed work.

Reply:
I never meant to do all of the `extra_specs` definitions that we would want.
But the ones that we touch anyway (`scs:name-v1`, `scs-name-v2`, `scs:cpu-type` and `scs:diskN-type`) feel half-defined, so let's have those complete. And do all the others in a future step.

Reply:
Yeah, that comment of mine is obsolete by now. I understand your reasoning, and I don't oppose it.